### ISSUE TITLE
rgb with only two params incorrectly parse color

### ISSUE THREAD

**Issue by gerdasi** [Qix-/color#234]
Hey!

I'm using `color` package and have some local tests for invalid user input check. After recent release (`4.0.2`) couple of my tests started failing. 
Previously `rgb(255,2)` color was invalid and now it returns `[25, 5, 2]`. 
The same goes for `rgba(255,0)`  - it returns `[25, 5, 0]`.

Is this expected behaviour? As far as I know, it should be invalid syntax 🤔 . 
Thanks! 🙏

**Comment by gerdasi** [comment]
Seems like it's not related to `color` package because downgrading versions didn't help.

**Comment by Qix-** [comment]
`rgb(255,2)` is not valid syntax, right. Can you show me exactly what you're using to instantiate `Color`? This might be a `color-string` bug as I just upgraded a number of things over there. Downgrading `color` probably wouldn't do much.

**Comment by mastertheblaster** [comment]
<img width="289" alt="Screenshot 2021-11-30 at 20 33 28" src="https://user-images.githubusercontent.com/10008149/144106940-28758fc3-4c25-4365-9239-b45c0afa7590.png">

can be reproduced with latest `npm install color`

**Comment by Qix-** [comment]
Yep, confirmed - this should be erroring. Regression introduced in `85e6f53831a6689a289e52766b49c8fafa6ec3eb` - I'll move this over to the `color-string` repository.
